addons/common/functions/fnc_getLocalUnits.sqf

```diff
@@ -0,0 +1,22 @@
++/*
+ * Author: dedmen  
+ * Gets localUnits array filtering out nullObjects.
+ * If you can handle null objects you can use the array array `ace_common_localUnits` directly.
+ * Should be equivalent to `allUnits select {local _x}`
+ *
+ * Arguments:
+ * Nothing
+ *
+ * Return Value:
+ * Array of local Units <ARRAY>
+ *
+ * Example:
+ * [] call ace_common_fnc_getLocalUnits
+ *
+ * Public: Yes
+ */
+
+//Remove null objects
+GVAR(localUnits) = GVAR(localUnits) - [objNull];
```

Comment:
We can check diag_frameNo against a global variable so we only remove null objects once a frame to get us a little more perf.
But I'm not sure if `GVAR(localUnits) = GVAR(localUnits) - [objNull];` is really that expensive that it's worth it. Needs profiling with atleast a hundred local units.

Reply:
Actually we only need to run this if there was a deletedEH in the last 2-3 frames. But I don't think it hurts.
Considering this will already be about a 100x improvement vs iterating through allUnits everytime.

Reply:
also to checking `diag_frameNo` objects are deleted in simulation cycle. Draw3D runs after that in the same frame. So that could cause Draw3D to still get null objects.